tests/network-tests/src/fixtures/workingGroups/SpendBudgetFixture.ts

```diff
@@ -9,6 +9,12 @@ import { ISubmittableResult } from '@polkadot/types/types/'
 import { Utils } from '../../utils'
 import { BudgetSpendingEventFieldsFragment, WorkingGroupFieldsFragment } from '../../graphql/generated/queries'
 
+export type VestingSchedule = {
```

Comment:
unused type, I think you mentioned integration tests will be updated in a separate PR?

Reply:
yes